packages/fuels-signers/src/wallet.rs

```diff
@@ -299,9 +299,9 @@ impl Wallet {
         Ok((tx.id().to_string(), receipts))
     }
 
-    /// Unconditionally transfers `amount` coins of type `asset_id` to
+    /// Unconditionally transfers `balance` of type `asset_id` to
     /// the contract at `to`.
-    /// Fails if amount for `asset_id` is larger than this wallet's spendable coins.
+    /// Fails if balance for `asset_id` is larger than this wallet's spendable coins.
```

Comment:
```suggestion
    /// Fails if balance for `asset_id` is larger than this wallet's spendable balance.
```

Reply:
Just for clarifying: my issue wasn't with the word "amount" per itself, but the fact that the docstring said `amount of _coins_` which is plain wrong. Feel free to use `amount` if you think it's better than balance, it just has to be super clear that what counts is the total balance (ie the total sum of the amounts of the various coins) and *not* the number of UTXOs (which is what was previously described).